### PR TITLE
Change GetVersion() timeout

### PR DESCRIPTION
**Pull request by koooge**
Hi there. Waiting time of `$ helm version`response from server is too long(300 sec). I think 5sec is better same as NewClient()

### CODE REVIEW

**Review by mattjmcnaughton — APPROVED** [review]
I'm not a maintainer but LGTM modulo my one comment :)

**Review by bacongobbler — CHANGES_REQUESTED** [review]
With this change, the timeout to connect to tiller is now hard-coded at 5 seconds and users cannot change this *at all* via `--tiller-connection-timeout`. I don't think that is the right approach we want to take.

**Comment by bacongobbler** [comment]
```
><> time ./bin/helm --tiller-connection-timeout=1 version
Client: &version.Version{SemVer:"v2.8+unreleased", GitCommit:"1b17b643fb278c9bb02ffc43de2501b0ddaca4a8", GitTreeState:"clean"}
Error: cannot connect to Tiller

real    0m5.068s
user    0m0.064s
sys     0m0.031s
```

**Comment by koooge** [comment]
I just want to get version (only client) and don't want to type long option.
In addition, `helm version` seems to be become to wait **30 sec** if tiller is not running on latest master. 😩 
I try to consider another approach about this issue.

```
$ time ./bin/helm version
Client: &version.Version{SemVer:"v2.8+unreleased", GitCommit:"be6eee18cb0f6d04cee8556fc758cccc052b1e8a", GitTreeState:"clean"}
Error: cannot connect to Tiller

real	0m35.112s
user	0m0.063s
sys	0m0.023s
```

**Comment by bacongobbler** [comment]
If you just want the client version you can use the --client-only option :)

**Pull request closed without merging**
